Add instructions to enable/disable Sippy from the dashboard (#12823)
cloudflare/cloudflare-docs: content/r2/data-migration/sippy.md

Before getting started, you will need:

- An existing R2 bucket. If you don't already have one, refer to [Create buckets](/r2/buckets/create-buckets/).
- [API credentials](/r2/data-migration/sippy/#create-credentials-for-storage-providers) for your source object storage bucket.
- (Optional) Cloudflare R2 Access Key ID and Secret Access Key with read and write permissions. For more information, refer to [Authentication](/r2/api/s3/tokens/).

### Enable Sippy via the Dashboard

1. From the Cloudflare dashboard, select **R2** from the sidebar.
2. Select the bucket you'd like to migrate objects to.

> 3. Switch to the **Settings** tab, then scroll down to the **Incremental migration** card.

Reviewer: ```suggestion
3. Switch to the **Settings** tab and scroll down to the **Incremental migration** card.
```

> 4. Press **Enable**, then enter details for the AWS / GCS bucket you'd like to migrate objects from. The credentials you enter must have permissions to read from this bucket. It is also recommended to scope your credentials to only allow reads from this bucket.

Reviewer: ```suggestion
4. Select **Enable** and enter details for the AWS / GCS bucket you'd like to migrate objects from. The credentials you enter must have permissions to read from this bucket. Cloudflare also recommends scoping your credentials to only allow reads from this bucket.
```

> 5. Press **Enable**.

Reviewer: ```suggestion
5. Select **Enable**.
```

- An existing R2 bucket. If you don’t already have one, refer to [Create buckets](/r2/buckets/create-buckets/).
- [API credentials](/r2/data-migration/sippy/#create-credentials-for-storage-providers) for your source object storage bucket.

> - (Optional) Cloudflare R2 Access Key ID and Secret Access Key with read and write permissions. For more information, refer to [Authentication](/r2/api/s3/tokens/).

Reviewer: Maybe update to (Wrangler only) instead of (Optional)?

### Enable Sippy via API

> The [API documentation](/api/operations/r2-put-bucket-sippy-config) documents required parameters and examples to enable Sippy. For information about getting started with the Cloudflare API, refer to [Make API calls](/fundamentals/api/how-to/make-api-calls/).

Reviewer: ```suggestion
For information on required parameters and examples of how to disable Sippy, refer to the [API documentation](/api/operations/r2-delete-bucket-sippy-config). For information about getting started with the Cloudflare API, refer to [Make API calls](/fundamentals/api/how-to/make-api-calls/).
```

{{<Aside type="note">}}

If your bucket is setup with [jurisdictional restrictions](/r2/reference/data-location/#jurisdictional-restrictions), you will need to pass a `cf-r2-jurisdiction` request header with that jurisdiction. For example, `cf-r2-jurisdiction: eu`.

{{</Aside>}}

## Disable Sippy on your R2 bucket

### Dashboard

1. From the Cloudflare dashboard, select **R2** from the sidebar.
2. Select the bucket you'd like to disable Sippy for.
3. Switch to the **Settings** tab, then scroll down to the **Incremental migration** card.

> 5. Press **Disable**.

Reviewer: ```suggestion
5. Select **Disable**.
```

```

### API

> The [API documentation](/api/operations/r2-delete-bucket-sippy-config) documents required parameters and examples to disable Sippy.

Reviewer: ```suggestion
For more information on required parameters and examples of how to disable Sippy, refer to the [API documentation](/api/operations/r2-delete-bucket-sippy-config).
```
